ui/src/components/EntityTable/EntityTable.scss

```diff
@@ -8,4 +8,30 @@
   .updating {
     opacity: 0.4;
   }
+}
+
+//small screen
```

Comment:
This looks messy. Do we need to do it separately for medium and small? Why do we have two levels of invisibility going on?